src/main/java/org/apache/maven/plugins/dependency/analyze/AbstractAnalyzeMojo.java

```diff
@@ -222,6 +222,14 @@
     @Parameter
     private final String[] ignoredUnusedDeclaredDependencies = new String[0];
 
+    /**
+     * List of project packaging that will be ignored.
+     *
+     * @since 3.2.1
```

Comment:
IMHO it will be new feature so according to semver we should bump minor version - what do your think?